### DISCUSSION TITLE
No hope running PascalABC.NET Linux on FreeBSD

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#3189] [General]
Even though it has Linux in its name, I think it's supposed to run wherever Mono is available. I installed Mono on FreeBSD. But when I try to run it via Mono, I always face obscure exceptions that prevented the program from starting.

**Comment by SunSerega** [comment]
The non-Linux version also runs on Mono - but many small things are currently not platform agnostic, so the non-Linux version would only work if you run Mono on Windows.

So the Linux version IS supposed to be compatible. But I reckon you are the first to try and run it on FreeBSD.
You can at least share the errors, but don't expect this to receive priority from the devs - so I recommend you look into the code yourself.
